certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG

```diff
@@ -31,6 +33,11 @@
 *   OPTION4: Renew certificate (roll over)
 *   OPTION5: Disburse funds by check
 *   OPTION6: Suspend certificate
+*
+*   OPTION 2, if selected by the member, it results in the member being
```

Comment:
Looks good to me, I would just remove the word "it" on line 37 or correct the grammar in another way.

Reply:
Yep, I would agree.... but I was asked to specifically add it in so..?   :-)

Reply:
alright! :)

Reply:
Sorry I got behind. 
Yes; the change looks good 
to let the person setting up the PO config know what Option 2 does,  
as the UX just tells the member to contact the CU to change the certificate term when they select that in Banno.